[2022-07-20 04:45:29,368] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_xcom_test.get_data scheduled__2022-07-19T00:00:00+00:00 [queued]>
[2022-07-20 04:45:29,411] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_xcom_test.get_data scheduled__2022-07-19T00:00:00+00:00 [queued]>
[2022-07-20 04:45:29,411] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-20 04:45:29,411] {taskinstance.py:1244} INFO - Starting attempt 1 of 6
[2022-07-20 04:45:29,412] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-20 04:45:29,462] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): get_data> on 2022-07-19 00:00:00+00:00
[2022-07-20 04:45:29,467] {standard_task_runner.py:52} INFO - Started process 92 to run task
[2022-07-20 04:45:29,477] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_xcom_test', 'get_data', 'scheduled__2022-07-19T00:00:00+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/postgres_xcom_test.py', '--cfg-path', '/tmp/tmpimoyy_ig', '--error-file', '/tmp/tmpnunm_fon']
[2022-07-20 04:45:29,478] {standard_task_runner.py:77} INFO - Job 73: Subtask get_data
[2022-07-20 04:45:29,680] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_xcom_test.get_data scheduled__2022-07-19T00:00:00+00:00 [running]> on host 0741836b2d3b
[2022-07-20 04:45:31,014] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kimuksung2
AIRFLOW_CTX_DAG_ID=postgres_xcom_test
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T00:00:00+00:00
[2022-07-20 04:45:31,015] {python.py:175} INFO - Done. Returned value was: {3: 5}
[2022-07-20 04:45:31,248] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=postgres_xcom_test, task_id=get_data, execution_date=20220719T000000, start_date=20220720T044529, end_date=20220720T044531
[2022-07-20 04:45:31,432] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-20 04:45:31,654] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-20 04:47:01,570] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_xcom_test.get_data scheduled__2022-07-19T00:00:00+00:00 [queued]>
[2022-07-20 04:47:01,591] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_xcom_test.get_data scheduled__2022-07-19T00:00:00+00:00 [queued]>
[2022-07-20 04:47:01,591] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-20 04:47:01,591] {taskinstance.py:1244} INFO - Starting attempt 1 of 6
[2022-07-20 04:47:01,591] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-20 04:47:01,622] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): get_data> on 2022-07-19 00:00:00+00:00
[2022-07-20 04:47:01,627] {standard_task_runner.py:52} INFO - Started process 146 to run task
[2022-07-20 04:47:01,632] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_xcom_test', 'get_data', 'scheduled__2022-07-19T00:00:00+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/postgres_xcom_test.py', '--cfg-path', '/tmp/tmpqsmypey2', '--error-file', '/tmp/tmpwg5dbxnz']
[2022-07-20 04:47:01,633] {standard_task_runner.py:77} INFO - Job 76: Subtask get_data
[2022-07-20 04:47:01,729] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_xcom_test.get_data scheduled__2022-07-19T00:00:00+00:00 [running]> on host 0741836b2d3b
[2022-07-20 04:47:01,819] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kimuksung2
AIRFLOW_CTX_DAG_ID=postgres_xcom_test
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T00:00:00+00:00
[2022-07-20 04:47:01,820] {python.py:175} INFO - Done. Returned value was: {3: 5}
[2022-07-20 04:47:01,862] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=postgres_xcom_test, task_id=get_data, execution_date=20220719T000000, start_date=20220720T044701, end_date=20220720T044701
[2022-07-20 04:47:02,092] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-20 04:47:02,156] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-20 05:00:29,557] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_xcom_test.get_data scheduled__2022-07-19T00:00:00+00:00 [queued]>
[2022-07-20 05:00:29,569] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_xcom_test.get_data scheduled__2022-07-19T00:00:00+00:00 [queued]>
[2022-07-20 05:00:29,570] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-20 05:00:29,570] {taskinstance.py:1244} INFO - Starting attempt 1 of 6
[2022-07-20 05:00:29,570] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-20 05:00:29,586] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): get_data> on 2022-07-19 00:00:00+00:00
[2022-07-20 05:00:29,590] {standard_task_runner.py:52} INFO - Started process 629 to run task
[2022-07-20 05:00:29,593] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_xcom_test', 'get_data', 'scheduled__2022-07-19T00:00:00+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/postgres_xcom_test.py', '--cfg-path', '/tmp/tmpi8vesigm', '--error-file', '/tmp/tmpsgc_qcj1']
[2022-07-20 05:00:29,593] {standard_task_runner.py:77} INFO - Job 79: Subtask get_data
[2022-07-20 05:00:29,649] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_xcom_test.get_data scheduled__2022-07-19T00:00:00+00:00 [running]> on host 0741836b2d3b
[2022-07-20 05:00:29,713] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kimuksung2
AIRFLOW_CTX_DAG_ID=postgres_xcom_test
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T00:00:00+00:00
[2022-07-20 05:00:29,714] {python.py:175} INFO - Done. Returned value was: {3: 5}
[2022-07-20 05:00:29,741] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=postgres_xcom_test, task_id=get_data, execution_date=20220719T000000, start_date=20220720T050029, end_date=20220720T050029
[2022-07-20 05:00:29,766] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-20 05:00:29,803] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-07-20 05:04:13,609] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_xcom_test.get_data scheduled__2022-07-19T00:00:00+00:00 [queued]>
[2022-07-20 05:04:13,623] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_xcom_test.get_data scheduled__2022-07-19T00:00:00+00:00 [queued]>
[2022-07-20 05:04:13,623] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-20 05:04:13,623] {taskinstance.py:1244} INFO - Starting attempt 1 of 6
[2022-07-20 05:04:13,623] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-20 05:04:13,639] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): get_data> on 2022-07-19 00:00:00+00:00
[2022-07-20 05:04:13,643] {standard_task_runner.py:52} INFO - Started process 773 to run task
[2022-07-20 05:04:13,646] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_xcom_test', 'get_data', 'scheduled__2022-07-19T00:00:00+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/postgres_xcom_test.py', '--cfg-path', '/tmp/tmp5osbbgo3', '--error-file', '/tmp/tmpiq8r_vvh']
[2022-07-20 05:04:13,647] {standard_task_runner.py:77} INFO - Job 82: Subtask get_data
[2022-07-20 05:04:13,711] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_xcom_test.get_data scheduled__2022-07-19T00:00:00+00:00 [running]> on host 0741836b2d3b
[2022-07-20 05:04:13,778] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kimuksung2
AIRFLOW_CTX_DAG_ID=postgres_xcom_test
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T00:00:00+00:00
[2022-07-20 05:04:13,779] {python.py:175} INFO - Done. Returned value was: {3: 5}
[2022-07-20 05:04:13,805] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=postgres_xcom_test, task_id=get_data, execution_date=20220719T000000, start_date=20220720T050413, end_date=20220720T050413
[2022-07-20 05:04:13,859] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-20 05:04:13,908] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
